src/components/PathwayExercise/CourseCompletion/LastCoursePage.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Typography, Container, Box, Button } from "@mui/material";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import { breakpoints } from "../../../theme/constant";
+import { interpolatePath, PATHS } from "../../../constant";
+import { useHistory } from "react-router-dom";
+import { useParams } from "react-router-dom";
+
+function LastCoursePage() {
+  const isActive = useMediaQuery("(max-width:" + breakpoints.values.sm + "px)");
+  const history = useHistory();
+  const params = useParams();
+  return (
+    <>
+      <Container maxWidth="lg" align="center">
+        <Box mt={8}>
+          <img src={require("../asset/specialdeals.svg")} alt="icon" />
+          <Typography variant="h6" mt={3} mb={4}>
+            Congratulations! You completed the track
+          </Typography>
+          <Button
+            onClick={() => {
+              history.push(
+                interpolatePath(PATHS.PATHWAY_COURSE, {
+                  pathwayId: params.pathwayId,
+                })
+              );
+            }}
+            variant="contained"
+          >
+            Return to Dashboard
+          </Button>
+        </Box>
```

Comment:
@Apekshahande

Reply:
Can you please explain me this ?

Reply:
@Apekshahande Sure, the problem is that the `<div>` with the back button is being taken out of the page flow because it's given fixed positioning.  So, if it's 64 pixels in height, the bottom 64 pixels of the page will get covered.  When the viewport (what you see in the browser window) is more than 64 pixels of the height of the `<body>` content, you won't notice this because it'll just be covering the extra space.  But when the browser window is smaller than the height of the content and you need to scroll vertically as shown in the picture, it'll cover those bottom 64 pixels with content.  In this case, the Return To Dashboard button gets covered.

So, a trick would be to give a container of the button 64 pixels of bottom margin such as the `Box`.  This way, the `<div>` with the Back still covers the bottom 64 pixels, but now it's covering the 64 pixel margin below the button.  So, the user is still able to see the entirety of the Return to Dashboard button which was once covered by scrolling to the bottom of the page.